gooddata-sdk/gooddata_sdk/catalog/declarative/analytics_layer.py

```diff
@@ -0,0 +1,155 @@
+# (C) 2022 GoodData Corporation
+from __future__ import annotations
+
+from typing import Any, Type, TypeVar
+
+from gooddata_metadata_client.model.declarative_analytical_dashboard import DeclarativeAnalyticalDashboard
+from gooddata_metadata_client.model.declarative_analytics_layer import DeclarativeAnalyticsLayer
+from gooddata_metadata_client.model.declarative_dashboard_plugin import DeclarativeDashboardPlugin
+from gooddata_metadata_client.model.declarative_filter_context import DeclarativeFilterContext
+from gooddata_metadata_client.model.declarative_metric import DeclarativeMetric
+from gooddata_metadata_client.model.declarative_visualization_object import DeclarativeVisualizationObject
+
+T = TypeVar("T", bound="CatalogAnalyticsBase")
+
+
+class CatalogDeclarativeAnalyticsLayer:
+    def __init__(
+        self,
+        analytical_dashboards: list[CatalogDeclarativeAnalyticalDashboard] = None,
```

Comment:
default [], please, we anyway have to inject e.g. analytical_dashboard key into the payload